### ISSUE TITLE
logging_target is not allowed during unattended installation but is used in code

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/framework#904]
```
Allowed keys:\\n - {1}'.format('\\n - '.join(errors), '\\n - '.join(expected_keys)))\r\nValueError: \r\nErrors found while verifying pre-configuration:\r\n - Key logging_target is not supported by OpenvStorage to be used in the pre-configuration JSON\r\n\r\nAllowed keys:\r\n - cluster_ip\r\n - config_mgmt_type\r\n - enable_heartbeats\r\n - external_config\r\n - master_ip\r\n - master_password\r\n - node_type\r\n - rdma\r\n", "msg": "MODULE FAILURE", "parsed": false}
```

used in code as seen here: https://github.com/openvstorage/framework/blob/master/ovs/lib/setup.py#L96

**Comment by wimpers** [comment]
If required earlier, please create a PR.

**Comment by khenderick** [comment]
Supported formats for `logging_target`:

In case of logging to console:

```
{'type': 'console'}
```

In case of redis logging:

```
{'type': 'redis',
 'queue': '<some_redis_queue>',  # can contain '{0}' which will be replaced by the logging source
 'host': '<redis ip address>',
 'port': <redis port>}
```

In case of logging to file

```
{'type': 'file'}
```

**Comment by khenderick** [comment]
Fixed by #1036, packaged in openvstorage-2.7.4-rev.4145.680473f

**Comment by JeffreyDevloo** [comment]
### Information
We are using unattended install on a daily basis. We did not encounter this error anymore for the past several months.
